--- udisks-1.0.5/configure.ac.orig	2014-03-11 16:37:31.471100389 +0100
+++ udisks-1.0.5/configure.ac	2014-03-11 16:38:54.587763572 +0100
@@ -3,12 +3,11 @@
 AC_PREREQ(2.59c)
 AC_INIT(udisks, 1.0.5, http://bugs.freedesktop.org/enter_bug.cgi?product=udisks)
 AM_INIT_AUTOMAKE(udisks, 1.0.5)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AM_MAINTAINER_MODE
 
 AC_ISC_POSIX
 AC_PROG_CC
-AM_PROG_CC_STDC
 AC_HEADER_STDC
 AM_PROG_LIBTOOL
 AC_PROG_MAKE_SET
